torch/export/_remove_effect_tokens_pass.py

```diff
@@ -44,6 +44,7 @@ def _remove_effect_tokens_from_graph_helper(
     output_node.args = (tuple(output_args[num_tokens:]),)
     for out_token in out_token_nodes:
         assert out_token.name in output_token_names
```

Comment:
can you help me add an assertion message here so that (hopefully) it is easier to debug in the future?